README.md

```diff
@@ -56,186 +50,416 @@ npm install @dojo/has
 
-These functions express structures that will be passed to the VDOM.
+#### Composing Widgets
 
-`v` creates nodes that represent DOM tags, e.g. `div`, `header` etc.
-This function allows Dojo 2 to manage lazy hyperscript creation and element caching.
+As mentioned, often widgets are composed of other widgets in their `render` output. This promotes widget reuse across an application (or multiple applications) and helps enables widget best practices.
 
- `w` creates Dojo 2 widgets or custom widget.
-This function provides support for lazy widget instantiation, instance management and caching.
+To compose widgets, we need to create `WNode`s and we can do this using the `w()` function from `@dojo/widget-core/d`.
 
-The `v` & `w` functions are available from the `@dojo/widget-core/d` package.
+Consider the previous `Hello` widget that we created:
 
 ```ts
-import { v, w } from '@dojo/widget-core/d';
+class App extends WidgetBase {
+	protected render(): DNode {
+		return v('div', [
+			w(Hello, { name: 'Bill' }),
+			w(Hello, { name: 'Bob' }),
+			w(Hello, { name: 'Flower pot men' })
+		]);
+	}
+}
 ```
 
-The argument and return types for `v` and `w` are available from `@dojo/widget-core/interfaces`, and are as follows:
+We can now use `App` with the `ProjectorMixin` to render the `Hello` widgets.
 
 ```ts
-import { DNode, HNode, WNode } from '@dojo/widget-core/interfaces';
+const Projector = ProjectorMixin(App);
+const projector = new Projector();
+
+projector.append();
 ```
 
-#### `v`
+**Note:** Widgets must return a single top level `DNode` from the `render` method, which is why the `Hello` widgets were wrapped in `div`.
 
-The following code creates an element with the specified `tag`
+#### Decomposing Widgets
 
-```ts
-v(tag: string): HNode[];
-```
+Splitting widgets into multiple smaller widgets is easy and helps to add extended functionality and promotes reuse.
 
-The following code renders an element with the `tag` and `children`.
+Consider the following `List` widget, which has a simple property interface of an array of items consisting of `content: string` and `highlighted: boolean`.
 
 ```ts
-v(tag: string, children: DNode[]): HNode[];
+interface ListProperties extends WidgetProperties {
+	items: {
+		id: string;
+		content: string;
+		highlighted: boolean;
+	};
+}
+
+class List extends WidgetBase<ListProperties> {
+	protected render() {
+		const { items } = this.properties;
+
+		return v('ul', { classes: { list: true } }, items.map((item) => {
+			const { id, highlighted, content } = item;
+			const classes = highlighted ? { highlighted: true } : { highlighted: false };
+
+			return v('li', { key: id, classes }, [ content ]);
+		});
+	}
+}
 ```
+The `List` widget does works as expected and displays the list in the browser but is hard to change, add functionality and reuse parts.
```

Comment:
"but is hard to change, add functionality and reuse parts." -> "but is difficult to reuse, modify, and/or extend"